osquery/tables/system/darwin/user_groups.mm

```diff
@@ -86,7 +141,24 @@ QueryData genGroups(QueryContext &context) {
   return results;
 }
 
-void setRow(Row &r, passwd *pwd) {
+void setRow(Row& r, passwd* pwd, boost::property_tree::ptree& tree) {
+  boost::optional<std::string> creationTime =
+      tree.get_optional<std::string>("creationTime");
+  boost::optional<std::string> failedLoginCount =
+      tree.get_optional<std::string>("failedLoginCount");
+  boost::optional<std::string> failedLoginTimestamp =
+      tree.get_optional<std::string>("failedLoginTimestamp");
+  boost::optional<std::string> passwordLastSetTime =
+      tree.get_optional<std::string>("passwordLastSetTime");
+
+  r["creation_time"] = creationTime ? creationTime.get() : DOUBLE(0);
```

Comment:
Does the truthy side of this ternary need a `DOUBLE()` around it? Similar comment on the next few.